app/lib/features/events/widgets/event_date_widget.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:acter/common/utils/utils.dart';
+import 'package:acter/features/events/event_utils/event_utils.dart';
+import 'package:acter/features/events/providers/event_providers.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart';
+import 'package:flutter/material.dart';
+
+class EventDateWidget extends StatelessWidget {
+  final CalendarEvent calendarEvent;
+  final double size;
+
+  const EventDateWidget({
+    super.key,
+    required this.calendarEvent,
+    this.size = 70,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return _buildEventDate(context);
+  }
+
+  Widget _buildEventDate(BuildContext context) {
+    final day = getDayFromDate(calendarEvent.utcStart());
+    final month = getMonthFromDate(calendarEvent.utcStart());
```

Comment:
bridging the SDK barrier is expensive, caching is always cheaper. These should probably be providers for an eventId.

Reply:
Agree but prefer to do in separate PR so that this big PR not get stretched out much.